package.json

```diff
@@ -6,7 +6,17 @@
   "scripts": {
     "prepare": "husky install"
   },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node-esm",
+      "js": "ts-node-esm"
+    }
+  },
   "devDependencies": {
+    "@types/express": "^4.17.20",
```

Comment:
@types/express and @types/node also only in the exercise where they're used. The -eslint ones should be here at the root, though.

Reply:
got rid of them from the root, will update per excerise going forward.